### ISSUE TITLE
Tool exiting with error

### ISSUE THREAD

**Issue by notmarshmllow** [trufflesecurity/trufflehog#310]
Tool exits with following error: 2022/04/07 14:55:46 [updater parent] prog returned error: exit status 1 while scanning target.

Debug Error Message:

```
DEBU[0152] Scanning complete. Scan time: 61.906127
DEBU[0152] scanned 26/31 repos
ERRO[0185] failed to clone repo                          exit_code=128 output="Cloning into '/tmp/trufflehog3964787350'...\nerror: 3557 bytes of body are still expected\nfatal: the remote end hung up unexpectedly\nfatal: early EOF\nfatal: index-pack failed\n" repo="https://github.com/testorg/Insights-Customer.git"
ERRO[0185] unable to clone repo (https://github.com/testorg/Insights-Customer.git), continuing  error="could not clone repo: https://github.com/testorg/Insights-Customer.git"
DEBU[0189] Scanning complete. Scan time: 11.235706
DEBU[0189] scanned 27/31 repos
ERRO[0192] failed to clone repo                          exit_code=128 output="Cloning into '/tmp/trufflehog3890337755'...\nerror: 629 bytes of body are still expected\nfatal: the remote end hung up unexpectedly\nfatal: early EOF\nfatal: index-pack failed\n" repo="https://github.com/testorg/Infrastructure.git"
ERRO[0192] unable to clone repo (https://github.com/testorg/Infrastructure.git), continuing  error="could not clone repo: https://github.com/testorg/Infrastructure.git"
ERRO[0193] failed to clone repo                          exit_code=128 output="Cloning into '/tmp/trufflehog1395488234'...\nerror: 7942 bytes of body are still expected\nfatal: the remote end hung up unexpectedly\nfatal: early EOF\nfatal: index-pack failed\n" repo="https://github.com/testorg/coreipc.git"
ERRO[0193] unable to clone repo (https://github.com/testorg/coreipc.git), continuing  error="could not clone repo: https://github.com/testorg/coreipc.git"
ERRO[0210] failed to clone repo                          exit_code=128 output="Cloning into '/tmp/trufflehog2524772988'...\nerror: 4838 bytes of body are still expected\nfatal: the remote end hung up unexpectedly\nfatal: early EOF\nfatal: index-pack failed\n" repo="https://github.com/testorg/Community.Activities.git"
ERRO[0210] unable to clone repo (https://github.com/testorg/Community.Activities.git), continuing  error="could not clone repo: https://github.com/testorg/Community.Activities.git"
DEBU[0211] scanned 123132 chunks
2022/04/07 16:16:48 [updater parent] prog returned error: exit status 1
2022/04/07 16:16:48 [updater parent] prog exited with 1
```

**Comment by dustin-decker** [comment]
related to #292

**Comment by dustin-decker** [comment]
This may be fixed in v3.2.2
